### ISSUE TITLE
Add non-interactive mode parameter for createReleaseBranch

### ISSUE THREAD

**Issue by topalavlad** [ru-fix/gradle-release-plugin#8]
Currently there is no option to run `createReleaseBranch` without human interaction. Even if the suggested version is correct, it still has to be confirmed.

For running from CI, I suggest adding a parameter that allows for `createReleaseBranch` to use the suggested version automatically. 

Do you think that makes sense or not really?

**Comment by swarmshine** [comment]
Hello, @topalavlad 
Thank you for feedback. 
In our teem we usually do not do major release branches from CI. 
But I think this feature makes sense.
Here is an option how we can do it:
add new property ru.fix.gradle.release.createDefaultReleaseBranch=true
that allows createReleaseBranch work in non-interactive way
In CI script we can declare:
```
gradle createReleaseBranch -Pru.fix.gradle.release.createDefaultReleaseBranch=true
```
What you think?

**Comment by swarmshine** [comment]
support for 
`gradle createReleaseBranch -Pru.fix.gradle.release.createDefaultReleaseBranch=true` 
implemented in  release 1.3.17

**Comment by topalavlad** [comment]
I think the non-interactive mode should also push the branch. Otherwise it's not usable because you have a new branch which the CI tool can tag, but the branch is never pushed out of the CI repository.
